wordpress-mobile/gutenberg-mobile
# Issue/1851 insert captions

Fixes #1851 

This PR adds the caption info to the Media object that is sent between the bridge and the JS code.

I also use this opportunity to add Stub image inserters for the Media Library in Android demo app to allow to test this functionality.

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/21611
WP-iOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/13903
WP-Android PR: https://github.com/wordpress-mobile/WordPress-Android/pull/11670

To test:
 - Start the demo app
 - Add an image block
 - Check that the predefined caption shows up for the Image

PR submission checklist:
